fiftyone/server/utils/json/encoder.py

```
@@ -30,7 +72,9 @@ def default(self, o):
 
 def dumps(obj: Any) -> str:
```

Ruff (0.15.6)

[warning] Dynamically typed expressions (typing.Any) are disallowed in `obj`

(ANN401)